org.fcitx.Fcitx5.Addon.McBopomofo.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: org.fcitx.Fcitx5.Addon.McBopomofo
+branch: stable
+runtime: org.fcitx.Fcitx5
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
+build-extension: true
+appstream-compose: false
+separate-locales: false
+build-options:
+  prefix: /app/addons/McBopomofo
+  prepend-pkg-config-path: "/app/addons/McBopomofo/lib/pkgconfig:/app/addons/McBopomofo/share/pkgconfig"
+cleanup:
+  - /bin
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - "*.la"
+  - /lib/cmake
+modules:
+  - name: fmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFMT_TEST=Off
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt
+        tag: 9.1.0
+  - name: fcitx5-mcbopomofo
+    buildsystem: cmake
```

Comment:
does cmake-ninja works? if it does it's better.

Reply:
It does. Updated now.